plugins/building-hacks.cpp

```diff
@@ -20,47 +20,56 @@
 #include "df/tile_building_occ.h"
 #include "df/building_drawbuffer.h"
 #include "df/general_ref_creaturest.h" // needed for power information storage
+#include "df/building_def_workshopst.h"
 #include "modules/Buildings.h"
 
 #include <map>
-
+#include <limits>
 using namespace DFHack;
 using namespace df::enums;
 
 DFHACK_PLUGIN("building-hacks");
 REQUIRE_GLOBAL(world);
 
+constexpr uint32_t invalid_tile = std::numeric_limits<uint32_t>::max();
```

Comment:
other parts of the code (including DF) uses 0 for "no tile". Would that make sense here, or does 0 have another meaning?